packages/docusaurus-theme-classic/src/theme/ThemeContext.js

```diff
@@ -9,8 +9,8 @@ import React from 'react';
 
 const ThemeContext = React.createContext({
   isDarkTheme: false,
-  setLightTheme: () => {},
-  setDarkTheme: () => {},
+  setLightTheme: 'UNINITIALIZED',
+  setDarkTheme: 'UNINITIALIZED',
```

Comment:
Not very fan of using fn: 'UNINITIALIZED' this way.

It's more common in React to have a context being initialized to undefined/null by default, and have a hook to read that context doing the null-check.

Can be seen here:
https://kentcdodds.com/blog/how-to-use-react-context-effectively#typescript--flow
https://medium.com/@dai_shi/a-thought-on-react-context-default-value-fb3283cb5788``